packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -389,10 +392,19 @@ export const makeVaultManager = (
     }
   };
 
+  const publicFacet = Far('collateral manager', {
+    makeVaultInvitation: () => zcf.makeInvitation(makeVaultKit, 'MakeVault'),
+    getNotifier: () => assetNotifer,
```

Comment:
wonder if we can collapse "collateral" and "asset" terms into one. "assetNotifier" was chosen because it's about the collateral and also the debt. Which implies "collateral manager" is doing more than managing collateral per se.

Reply:
Not well, since these are all assets, but what distinguishes here is that it is specific to a collateral type.